BackofficeBundle/Resources/public/coffee/page/node/NodeRouter.coffee

```diff
@@ -0,0 +1,48 @@
+###*
+ * Router node
+###
+((router) ->
+
+  ###*
+   * show node
+  ###
+  router.route 'node/show/:nodeId', 'showNode', (nodeId) ->
+    @showNodeAction(nodeId)
+
+  ###*
+   * show node with language
+  ###
+  router.route 'node/show/:nodeId/:language', 'showNodeWithLanguage', (nodeId, language) ->
+    @showNodeAction(nodeId, language)
+    return
+
+  ###*
+   * show node with language and version
+  ###
+  router.route 'node/show/:nodeId/:language/:version', 'showNodeWithLanguageAndVersion', (nodeId, language, version) ->
+    @showNodeAction(nodeId, language, version)
+    return
+
+  ###*
+   * load node and display node view
+  ###
+  router.showNodeAction = (nodeId, language = null, version = null)->
+    @initDisplayRouteChanges '#nav-node-' + nodeId
+    url = $('#nav-node-' + nodeId).data('url')
+    url = url + '?language=' + language if language?
```

Comment:
But language is get parameter not a parameter of the route open_orchestra_api_node_show. i agree that language should be a route parameter but i think isn't the goal of this pull request